neuralmonkey/train.py

```diff
@@ -183,6 +186,20 @@ def main() -> None:
         log(str(exc), color='red')
         exit(1)
 
+    if cfg.model.visualize_embeddings:
+
+        tb_projector = projector.ProjectorConfig()
+
+        for sequence in cfg.model.visualize_embeddings:
+            # TODO this check should be done when abstract class of embedded
+            # sequences will be created, not only EmbeddedFactorSequence
```

Comment:
Ne abstrakní třída pro embedlý sekvence, ale abstrakní class pro něco, co má schopnost embedovat a tedy i vizualizovat to embedování (tj. potřebuje vocabularies, embedovací matice a bude mít tu metodu, kterou teď @tomkocmi přidal do `EmbeddedSequence`). Jak jinak bys pak chtěl vizualizovat embedingy z rekurentního dekodéru?